code/views/new_queue.erb

```diff
@@ -45,6 +45,26 @@
 </div>
 
 <div class="field">
+  <label class="field_label" for="queue_blocking">Message blocking</label>
+  <label for="queue_blocking">blocking</label>
+  <input type="radio" name="queue[blocking]" value="yes" />
+  <label for="queue_blocking">no-blocking</label>
+  <input type="radio" name="queue[blocking]" value="no" checked />
+</div>
+
+<div class="field">
+  <label class="field_label" for="queue_blocking_param1"></label>
+  <label for="queue_blocking_param1">param1</label>
+  <input type="checkbox" name="queue[blocking_param1]" value="1" checked/>
```

Comment:
Default should be unchecked, since checking a param but turning off the feature is contradictory.